Backend/Controllers/Activities.cs

```diff
@@ -73,5 +73,108 @@ public async Task<IActionResult> DeleteActivity(uint id)
 
             return NoContent();
         }
+
+        // PATCH: api/activities/5/name
+        /// <summary>
+        /// Updates an activities name.
+        /// </summary>
+        /// <param name="id">The id of the activity to update.</param>
+        /// <param name="newName">The new name of the activity.</param>
+        /// <returns>No content.</returns>
+        [HttpPatch("{id}/name")]
+        public async Task<IActionResult> PatchActivityName(uint id, string newName)
+        {
+            Activity? activity = await db.Activities.FindAsync(id);
+            if (activity == null) return NotFound();
+
+            activity.Name = newName;
+
+            await db.SaveChangesAsync();
+
+            return NoContent();
+        }
+
+        // PATCH: api/activities/5/description
+        /// <summary>
+        /// Updates an activities description.
+        /// </summary>
+        /// <param name="id">The id of the activity to update.</param>
+        /// <param name="newDescription">The new description of the activity.</param>
+        /// <returns>No content.</returns>
+        [HttpPatch("{id}/description")]
+        public async Task<IActionResult> PatchActivityDescription(uint id, string newDescription)
+        {
+            Activity? activity = await db.Activities.FindAsync(id);
+            if (activity == null) return NotFound();
+
+            activity.Description = newDescription;
+
+            await db.SaveChangesAsync();
+
+            return NoContent();
+        }
+
+        // PATCH: api/activities/5/datetimestart
+        /// <summary>
+        /// Updates an activities start date and time.
```

Comment:
Can all these `PATCH` endpoints be combined into one? You accept a generic JSON object instead of a specific type, see if a field is specified, and if it is you find the field of the databse entry to update and overwrite it. The reason I think generic JSON works better than a static type (such as ActivityPatchDTO) works better, is because if a field is (or would become) nullable, then you cannot differntiate between a field in the DTO being null because it has not been specified, or because it must become null.